### PR TITLE
Fix iOS / Android CMake builds

### PR DESCRIPTION
**Pull request by juan-lunarg**
closes https://github.com/KhronosGroup/SPIRV-Tools/issues/4437

### CODE REVIEW

File: .github/workflows/cmake.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+name: cmake
+permissions:
+  contents: read
+
+on: [push, pull_request, workflow_dispatch]
```
Comment (juan-lunarg) [inline .github/workflows/cmake.yml:5]: Allows easy testing on forks.

**Comment by juan-lunarg** [comment]
> Do you know why the new CIs failed? I don't want to merge if they will start to fail for everyone else.

I couldn't figure out why the new CI failed even on my fork. I've removed the new CI from this PR

**Review by s-perron — APPROVED** [review]

**Comment by juan-lunarg** [comment]
> I couldn't figure out why the new CI failed even on my fork. I've removed the new CI from this PR

We have integration testing for iOS / Android in the vulkan validation layers and glslang. Which will do for now.

**Comment by juan-lunarg** [comment]
> We have integration testing for iOS / Android in the vulkan validation layers and glslang. Which will do for now.

Never mind I got CI working for this repo @s-perron 
https://github.com/KhronosGroup/SPIRV-Tools/pull/5490

**Pull request merged**